zulipterminal/model.py

```diff
@@ -634,7 +638,8 @@ def toggle_stream_muted_status(self, stream_id: int) -> bool:
         return response['result'] == 'success'
 
     def is_pinned_stream(self, stream_id: int) -> bool:
-        return stream_id in list(stream[1] for stream in self.pinned_streams)
+        return stream_id in list(stream['id']
+                                 for stream in self.pinned_streams)
```

Comment:
A regular list comprehension (not list constructor with generator expression) may simplify this onto one line?